src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -69,7 +70,7 @@ export class ApiKeyAuthenticator {
 
     return tryAuthenticate()
       .then((response: IResponse) => {
-        const body = response.body;
+        const body = response.body as IHash;
```

Comment:
Something like this: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#object-type